skill_markII_audio_receiver/__init__.py

```diff
@@ -205,11 +241,21 @@ def rename_spotify_intent(self, _) -> None:
             try:
                 self.renaming_spotify = False
                 self.log.debug(f"Renaming Raspotify device to {new_name}")
-                self.bus.emit(Message("neon.phal.plugin.audio.receiver.set.raspotify.name", {"name": new_name}))
+                self.bus.emit(
+                    Message(
```

Comment:
`message.forward` should be used here to preserve context (session, skill_id, etc)
```python
message.forward("neon.phal.plugin/audio.receiver.set.raspotify.name", {"name": new_name})
```